test/Microsoft.AspNet.Mvc.FunctionalTests/RemoteAttributeValidationTest.cs

```diff
@@ -23,12 +23,12 @@ public RemoteAttributeValidationTest(MvcTestFixture<ValidationWebSite.Startup> f
         public HttpClient Client { get; }
 
         [Theory]
-        [InlineData("Aria", "/Aria")]
+        [InlineData("Area1", "/Area1")]
```

Comment:
The reason I did this was that we are moving files from different websites to BasicWebSite and thought that the structure of BasicWebSite should look in a manner which goes well with other tests targeting this website. Also why `Aria` anyway? :smile: The structure of areas was also inconsistent, like we named the value in the attribute as `Aria` but the folder as `MyArea`